gpu/src/Queue.zig

```diff
@@ -67,6 +67,9 @@ pub inline fn writeTexture(
     data_layout: *const Texture.DataLayout,
     write_size: *const Extent3D,
 ) void {
+    if (@TypeOf(data) != []const u8) {
```

Comment:
Nice! I think there are more places than just here that we need to update - though.

* `Queue.zig` `fn writeBuffer`
* `CommandEncoder.zig` `fn writeBuffer`

Reply:
yeah im aware. will update this soon

Reply:
also this seems wrong (check CI log)

Reply:
It looks like `std.meta.isSliceOf(T)` might do what we want https://sourcegraph.com/github.com/ziglang/zig/-/blob/lib/std/meta/trait.zig?L142:7

Reply:
yeah make sense :+1: